cmake/sdlchecks.cmake

```
@@ -739,6 +739,22 @@ macro(CheckOpenGLES)
   endif()
 endmacro()
 
+# Requires:
+# - EGL
+macro(CheckQNXScreen)
+  if(QNX AND HAVE_OPENGL_EGL)
+    check_c_source_compiles("
+        #include <screen/screen.h>
+        int main (int argc, char** argv) { return 0; }" HAVE_QNX_SCREEN)
+    if(HAVE_QNX_SCREEN)
+      set(SDL_VIDEO_DRIVER_QNX 1)
+      file(GLOB QNX_VIDEO_SOURCES ${SDL3_SOURCE_DIR}/src/video/qnx/*.c)
+      list(APPEND SOURCE_FILES ${QNX_VIDEO_SOURCES})
+      list(APPEND EXTRA_LIBS screen EGL)
```

https://github.com/libsdl-org/SDL/pull/7017 changed this
```suggestion
      list(APPEND SDL_EXTRA_LIBS screen EGL)
```

---

Done. thanks.